src/lib/requests/thirdpartyRequests.js

```diff
@@ -2,50 +2,21 @@
 
 const BaseRequests = require('./baseRequests');
 
-
-/**
- * @class ThirdpartyRequests
- * @description Client library for making outbound Mojaloop requests
- *   for 3rd party functions (e.g. PISP use cases)
- */
 class ThirdpartyRequests extends BaseRequests {
+    async putConsents(consentId, consentBody, destParticipantId) {
```

Comment:
does this not take in the request headers? which would contain some info about the source/prev destination?

Reply:
The destination will be set with `destParticipantId`, and the source is configured by the config object in the `constructor()`